index.html

```diff
@@ -4813,13 +4813,45 @@ <h2>Contexts, Types, and Credential Schemas</h2>
       <section class="informative">
         <h3>Base Context</h3>
           <p>
-The base context, located at
-<code>https://www.w3.org/ns/credentials/v2</code> with a SHA-256 digest of
+The base context, located at <code>https://www.w3.org/ns/credentials/v2</code>
+with a SHA-256 digest of
 <strong><code>944167aaabd904ea9e35c98fd7e8794eb6dd42ae4666b036b171e87fc34cc7cc</code></strong>,
 can be used to implement a local cached copy. It is possible to confirm the
 SHA-256 digest by running the following command from a modern Unix command
-interface line: `curl -s https://www.w3.org/ns/credentials/v2 | sha256sum`. For
-convenience, the base context is also provided below.
+interface line: `curl -s https://www.w3.org/ns/credentials/v2 | sha256sum`. It
```

Comment:
`sha256sum` seems less desirable than using openssl...

Reply:
@OR13 — Suggest the alternative command?